test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,316 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and aten operators."""
+import torch
+from torch.testing._internal import (
+    common_device_type,
+    common_methods_invocations,
+    common_utils,
+)
+
+# Ops and the dtypes to be tested for consistency
+# TODO(justinchuby): We need a way to annotate symbolic functions with their supported dtypes
+ALLOWLIST_OP = {
+    "__getitem__": frozenset(["b8", "f16", "f32", "i16", "i32", "i64", "u8"]),
```

Comment:
Is there a place to find a complete operators set in this format?

Reply:
I think that's possible. Is there a user experience you have in mind?

Reply:
The discussion for first question can be carried on in a later comment I made at the same line. Basically how tight it links with actual torch aten library.

For second question, I'm thinking about the polished log of this test as a more accurate version for the supported/unsupported aten ops doc page.

Reply:
1. SG
2. We should have a nice matrix on that soon leveraging opinfo.